tests/framework/e2e/etcdctl.go

```diff
@@ -24,14 +24,16 @@ import (
 )
 
 type EtcdctlV3 struct {
-	cfg       *EtcdProcessClusterConfig
-	endpoints []string
+	cfg        *EtcdProcessClusterConfig
+	endpoints  []string
+	runningCfg []*EtcdServerProcessConfig
 }
 
-func NewEtcdctl(cfg *EtcdProcessClusterConfig, endpoints []string) *EtcdctlV3 {
+func NewEtcdctl(cfg *EtcdProcessClusterConfig, endpoints []string, runningCfg []*EtcdServerProcessConfig) *EtcdctlV3 {
 	return &EtcdctlV3{
-		cfg:       cfg,
-		endpoints: endpoints,
+		cfg:        cfg,
+		endpoints:  endpoints,
+		runningCfg: runningCfg,
```

Comment:
I don't think `[]*EtcdServerProcessConfig` is needed in the EtcdctlV3 now.

Reply:
You are right. it's not needed now.
I keep this for `etcdutl` in the furture.  I will delete it for  your suggestion.

Reply:
I don't think we should add this code just in case we need it. We might take different approach and not use this code at all.
Let's just remove it.

Reply:
Removed.